synapseclient/client.py

```diff
@@ -2358,22 +2359,24 @@ def _getUserbyPrincipalIdOrName(self, principalId: str = None):
                 "Unknown Synapse user (%s).  %s." % (principalId, supplementalMessage)
             )
 
-    @tracer.start_as_current_span("Synapse::getPermissions")
-    def getPermissions(
+    @tracer.start_as_current_span("Synapse::get_acl")
+    def get_acl(
         self,
         entity: Union[Entity, Evaluation, str, collections.abc.Mapping],
         principalId: str = None,
```

Comment:
Since `get_acl` is a new function let's also swap out the arugments for snake_case as well.